### ISSUE TITLE
r2c.flask failing to decode JSON output

### ISSUE THREAD

**Issue by arashd** [returntocorp/bento#229]
## Overview
> Ran bento on https://github.com/civicsoft/ieddit and r2c.flask failed with a JSON decode error.

## Current Behavior
```
$ bento check
Running Bento checks...

r2c.bandit․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․|■■■■■■■■■■| 00:01, 🍱 Done
r2c.flake8․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․|■■■■■■■■■■| 00:01, 🍱 Done
r2c.flask․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․|■■■■□□□□□□| 00:01, 🍤 Running
r2c.requests․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․|■■■■■■■■■■| 00:01, 🍱 Done

✘ Error while running r2c.flask: ("Could not parse output of 'r2c.flask':\n",
JSONDecodeError('Expecting value: line 1 column 1 (char 0)'))
-------------------------------------------------------------------------------------------------
This may be due to a corrupted tool installation. You might be able to fix this issue by running:

  bento init --clean

You can also view full details of this error in `/Users/arashdelijani/.bento/last.log`.
-------------------------------------------------------------------------------------------------

⚠ 95 finding(s) in 1.39 s
◦ To suppress all findings, run $ bento archive.
```

## Steps to Reproduce

1. `git clone git@github.com:civicsoft/ieddit.git`
2. `cd ieddit`
3. `bento init`

## Additional Context
Bento version (run `bento --version`):
`bento/0.7.0`
Operation system (e.g. Ubuntu 18.04):
MacOS Catalina

**Comment by minusworld** [comment]
Stack trace from last.log:
```
"""
Traceback (most recent call last):
  File "/usr/local/Cellar/python/3.7.5/Frameworks/Python.framework/Versions/3.7/lib/python3.7/multiprocessing/pool.py", line 121, in worker
    result = (True, func(*args, **kwds))
  File "/Users/grayson/sandbox/ieddit/.bento/flask/lib/python3.7/site-packages/flake8/checker.py", line 666, in _run_checks
    return checker.run_checks()
  File "/Users/grayson/sandbox/ieddit/.bento/flask/lib/python3.7/site-packages/flake8/checker.py", line 598, in run_checks
    self.run_ast_checks()
  File "/Users/grayson/sandbox/ieddit/.bento/flask/lib/python3.7/site-packages/flake8/checker.py", line 502, in run_ast_checks
    for (line_number, offset, text, check) in runner:
  File "/Users/grayson/sandbox/ieddit/.bento/flask/lib/python3.7/site-packages/flake8_flask/main.py", line 46, in run
    visitor.visit(self.tree)
  File "/usr/local/Cellar/python/3.7.5/Frameworks/Python.framework/Versions/3.7/lib/python3.7/ast.py", line 271, in visit
    return visitor(node)
  File "/usr/local/Cellar/python/3.7.5/Frameworks/Python.framework/Versions/3.7/lib/python3.7/ast.py", line 279, in generic_visit
    self.visit(item)
  File "/usr/local/Cellar/python/3.7.5/Frameworks/Python.framework/Versions/3.7/lib/python3.7/ast.py", line 271, in visit
    return visitor(node)
  File "/Users/grayson/sandbox/ieddit/.bento/flask/lib/python3.7/site-packages/flake8_flask/upsell_blueprint.py", line 97, in visit_FunctionDef
    route_prefix = os.path.split(route)[0]
  File "/Users/grayson/sandbox/ieddit/.bento/flask/bin/../lib/python3.7/posixpath.py", line 107, in split
    p = os.fspath(p)
TypeError: expected str, bytes or os.PathLike object, not NoneType
"""
```

**Comment by minusworld** [comment]
Fails on `ieddit.py`. Investigating...

**Comment by ulziibay** [comment]
I am happy to address this as I am oncall and I am working on https://github.com/returntocorp/bento-core/issues/348

**Comment by ulziibay** [comment]
We close this once https://github.com/returntocorp/bento-core/pull/516 is merged.

**Comment by pabloest** [comment]
I am running into something that seems similar to this issue.
However, I also get `JSONDecodeError` on Flake8, r2c.click, and others:
```
Running Bento checks...

bandit․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․|■■■■■■■■■■| 00:00, 🍱 Done
flake8․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․|■■■■■■■■■■| 00:00, 🍱 Done
r2c.click․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․|■■■■■■■■■■| 00:00, 🍱 Done
r2c.flask․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․|■■■■■■■■■■| 00:00, 🍱 Done
r2c.requests․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․․|■■■■■■■■■■| 00:00, 🍱 Done

✘ Error while running bandit: [Errno 2] No such file or directory: 'bandit':
'bandit'
-------------------------------------------------------------------------------------------------
This may be due to a corrupted tool installation. You might be able to fix this issue by running:

  bento init --clean

You can also view full details of this error in `/Users/pabloestrada/.bento/last.log`.
-------------------------------------------------------------------------------------------------

✘ Error while running flake8: ("Could not parse output of 'flake8':\n",
JSONDecodeError('Expecting value: line 1 column 1 (char 0)'))
-------------------------------------------------------------------------------------------------
This may be due to a corrupted tool installation. You might be able to fix this issue by running:

  bento init --clean

You can also view full details of this error in `/Users/pabloestrada/.bento/last.log`.
-------------------------------------------------------------------------------------------------

✘ Error while running r2c.click: ("Could not parse output of 'r2c.click':\n",
JSONDecodeError('Expecting value: line 1 column 1 (char 0)'))
-------------------------------------------------------------------------------------------------
This may be due to a corrupted tool installation. You might be able to fix this issue by running:

  bento init --clean

You can also view full details of this error in `/Users/pabloestrada/.bento/last.log`.
-------------------------------------------------------------------------------------------------

✘ Error while running r2c.flask: ("Could not parse output of 'r2c.flask':\n",
JSONDecodeError('Expecting value: line 1 column 1 (char 0)'))
-------------------------------------------------------------------------------------------------
This may be due to a corrupted tool installation. You might be able to fix this issue by running:

  bento init --clean

You can also view full details of this error in `/Users/pabloestrada/.bento/last.log`.
-------------------------------------------------------------------------------------------------

✘ Error while running r2c.requests: ("Could not parse output of
'r2c.requests':\n", JSONDecodeError('Expecting value: line 1 column 1 (char
0)'))
-------------------------------------------------------------------------------------------------
This may be due to a corrupted tool installation. You might be able to fix this issue by running:

  bento init --clean

You can also view full details of this error in `/Users/pabloestrada/.bento/last.log`.
-------------------------------------------------------------------------------------------------

✔ 0 findings in 6.90 s
```

I followed the suggestion to initialize --clean and I am running `bento check .` at the root project directory of this repo: git@github.com:pallets/flask.git.

I’ll reopen this issue so we can track it, feel free to close if the PR @ulziibay mentioned fixes the issue in the tools other than r2c.flask.

**Comment by dlukeomalley** [comment]
Thank you @pabloest! This is a regression that has been fixed for versions > 0.8.0b3.
